examples/http-client-endpoint/http_client_endpoint.bal

```diff
@@ -80,3 +51,18 @@ public function main() {
         log:printError("Error when calling the backend", err = response);
     }
 }
+
+//Function to handle response received from remote HTTP endpoint.
+function handleResponse(http:Response|error response) {
+    if (response is http:Response) {
+        var msg = response.getJsonPayload();
+        if (msg is json) {
+            // Print the received `json` response.
+            log:printInfo(msg.toString());
```

Comment:
Since this is a main program, shouldn't we transform these logs to io:prints? IIRC logs are used for services mainly

Reply:
@chamil321 : Yes. But we need to stick to either log or io IMO. But in this example, both have used in main method. (io for info messages and log for error messages) Since we need to log the err in the error messages logs seems easier than io. WDYT?

Reply:
@chamil321 Converted all logs into io package.